css/index.css

```diff
@@ -175,19 +175,109 @@ footer {
 }
 
 
+/* media queries */
+
+/* tablet*/
+
+
+@media(max-width: 768px){
+    .container{
+        width: 100%;
+        margin: 10px 200px 100px 10px;
+
+    }
+
+    header {
+        display: flex;
+        padding: 0 10px 0 1;0px;
+        width:90%;
+    }
+    
+  .middle-img{
+      width:100%;
+      padding-right: 7px;
+      
+  }
+
+  .cta{
+      display: flex;
+     justify-content: space-around;
+     width:99%;
+     height: 385px;
+
+  }
+  .cta-tex{
+      margin-right:0px;
+      border-right:0px
+  }
+}
 
-@media(max-width:1024px){
 
-body {
-    line-height: 1;
-    background: black;
-}
+@media(max-width:400px){
+
+    
+    .conatiner{
+        width:100%;
+        margin-left:0;
+
+    }
+    header{
+        display: flex;
+        flex-direction:column;
+        margin: 0px;
+        
+        width:100%;
 
+    }
 
+    header nav{
+    display: flex;
+    flex-direction: column;
+    width:100%;
+    padding:1px;
    
-.container {
-    width: 1024px;
-    height: 1536;
-    margin: auto;
-    background: white;
-}
\ No newline at end of file
+    margin-top:0px
+    }
+   nav a {
+       display:inline;
+       text-align: center;
+       padding-top:10px;
+       border-bottom: 2px solid black;
+       
+       width:100%;
+       height: 40px;
+      
+       
+   
+   }
+
+   .logo{
+       order:-1;
+       padding:5px;
+       border-bottom: 2px solid black;
+       width:100%;
+       margin-top: 5px;
+       
+       
+
+   }
+
+
+    .cta{
+        display:flex;
+        flex-direction: column;
+        flex-wrap:wrap;
+        height:800px;
+        padding-right:5px
+     
+    }
+
+    .cta-text{
+        padding-left:50px;
+    }   
+
+    footer{
+        margin-top:70px
+    }
+    
```

Comment:
Where's the adjustments for `top-content` and `bottom-content`?
![top-content bottom-content flexing](https://user-images.githubusercontent.com/12449668/37169133-49a24a30-22c4-11e8-9aa7-fdaa878c6b3b.png)